client/odie/context/use-load-previous-chat.ts

```diff
@@ -5,10 +5,9 @@ import { getOdieInitialMessage } from './get-odie-initial-message';
 
 export const useLoadPreviousChat = (
 	botNameSlug: OdieAllowedBots,
-	chatId: number | null | undefined,
-	onChatLoaded?: () => void
+	chatId: number | null | undefined
 ) => {
-	const { data: existingChat } = useOdieGetChat( botNameSlug, chatId );
+	const { data: existingChat } = useOdieGetChat( botNameSlug, chatId, 1, 30 );
```

Comment:
This seems to always be loading the chat with the most recent 30 messages (any messages before this wouldn't get returned). We would need to keep the page number in state and increment it once the user scrolls all the way to the top to load the next page of results? 

I changed the limit per page to a smaller value of 4 for testing. It seems that the initial prompt always shows up as the first message as well, which should probably be hidden if there are more messages to load.

Reply:
I added it partially as a work around, I think limiting the chat to 30 is enough for a good conversation. We might improve this on a second iteration better, rather than work now on an infinite scroll hook?